horovod/common/operations.h

```diff
@@ -44,6 +44,11 @@ namespace common {
 #define HOROVOD_HIERARCHICAL_ALLGATHER "HOROVOD_HIERARCHICAL_ALLGATHER"
 #define HOROVOD_CACHE_CAPACITY "HOROVOD_CACHE_CAPACITY"
 #define HOROVOD_MLSL_BGT_AFFINITY "HOROVOD_MLSL_BGT_AFFINITY"
+#define HOROVOD_CPU_RUNTIME "HOROVOD_CPU_OPERATIONS"
```

Comment:
Let's keep this `HOROVOD_CPU_OPERATIONS` and rename the macro to `HOROVOD_CPU_OPERATIONS_DEFAULT`?

Reply:
Fixed.